app/models/rental.rb

```diff
@@ -3,4 +3,6 @@ class Rental < ActiveRecord::Base
   has_many :rental_items
   has_many :items, through: :rental_items
   validates :user_id, :days_rented, :total_price, presence: true
+
+  default_scope -> { order('created_at DESC') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>`default_scope` expects a block as its sole argument.